db/schema.rb

```diff
@@ -10,24 +10,25 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_03_21_154839) do
+ActiveRecord::Schema[7.0].define(version: 2023_05_10_093121) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "citext"
   enable_extension "plpgsql"
 
   create_table "crime_applications", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
-    t.jsonb "application"
+    t.jsonb "submitted_details"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "status", default: "submitted", null: false
     t.datetime "submitted_at", precision: nil, default: -> { "CURRENT_TIMESTAMP" }
     t.datetime "returned_at", precision: nil
-    t.virtual "searchable_text", type: :tsvector, as: "((to_tsvector('english'::regconfig, (application #>> '{client_details,applicant,first_name}'::text[])) || to_tsvector('english'::regconfig, (application #>> '{client_details,applicant,last_name}'::text[]))) || to_tsvector('english'::regconfig, (application ->> 'reference'::text)))", stored: true
+    t.virtual "searchable_text", type: :tsvector, as: "((to_tsvector('english'::regconfig, (submitted_details #>> '{client_details,applicant,first_name}'::text[])) || to_tsvector('english'::regconfig, (submitted_details #>> '{client_details,applicant,last_name}'::text[]))) || to_tsvector('english'::regconfig, (submitted_details ->> 'reference'::text)))", stored: true
```

Comment:
I'm not sure all these virtual definitions will keep working just by renaming them.

The schema.rb is not applied on deployments, just new migrations are applied. In order for this to work the database would need to be recreated from the schema with `db:schema:load` which wipes the data.